db/schema.rb

```diff
@@ -600,7 +600,6 @@
     t.index ["work_id"], name: "index_work_view_stats_on_work_id"
   end
 
-  add_foreign_key "csv_imports", "users"
```

Comment:
Do you know why rails makes random changes to schema.rb? I've been getting those too, but not checking them in...

Reply:
I think it is because we have been adding fields? but here is someone else who had that problem

https://stackoverflow.com/questions/45494084/random-changes-in-schema-rb